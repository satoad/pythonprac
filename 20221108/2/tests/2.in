asdf
asjdopaodjpoj
1,2,3,4,5,6
(1,1), (2,2), (11,11)
asdadsdadasd
(1,2), (4,5), (9,8)
(1,1), (0, 0), (0, 3)