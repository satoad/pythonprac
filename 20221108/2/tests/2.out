Invalid Input
Invalid Input
Invalid Input
Not a triangle
Invalid Input
3.00
